maia/algo/part/cgns_registry/test/test_cgns_registry.py

```diff
@@ -1,5 +1,5 @@
-from pytest_mpi_check            import assert_mpi
```

Comment:
We usually dont use `assert_mpi` in maia since we can achieve same result with
if comm.Get_rank() == X:
  assert Y

can you remove the import and update the file ?

Reply:
ok rebase done